generator/osm_element.cpp

```diff
@@ -71,22 +69,15 @@ void OsmElement::AddTag(char const * key, char const * value)
   SKIP_KEY_BY_PREFIX("official_name");
 #undef SKIP_KEY_BY_PREFIX
 
-  std::string_view val(value);
-  strings::Trim(val);
-  m_tags.emplace_back(key, val);
-}
-
-void OsmElement::AddTag(std::string const & key, std::string const & value)
-{
-  AddTag(key.data(), value.data());
+  m_tags.emplace_back(key, value);
 }
 
-bool OsmElement::HasTag(std::string const & key) const
+bool OsmElement::HasTag(std::string_view const & key) const
 {
   return base::AnyOf(m_tags, [&](auto const & t) { return t.m_key == key; });
 }
 
-bool OsmElement::HasTag(std::string const & key, std::string const & value) const
+bool OsmElement::HasTag(std::string_view const & key, std::string_view const & value) const
```

Comment:
```suggestion
bool OsmElement::HasTag(std::string_view key, std::string_view value) const
```

Reply:
At least it is important for consistency. We don't want to have double standards, right? )